config/data_providers.yml

```
@@ -12,3 +12,7 @@ shared:
     name: DGFIP
     logo: dgfip.jpeg
     link: https://api.gouv.fr/producteurs/dgfip
+  minarm:
```

```suggestion
  ministere_des_armees:
```

quand même plus clair.